content/v1.14/cli/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+weight: 400
+title: Crossplane CLI
+description: "Documentation for the Crossplane command-line interface"
+---
+
+The Crossplane CLI helps simplify some development and administration aspects of
+Crossplane.
+
+The Crossplane CLI includes:
+* tools to build, install, update and push Crossplane Packages
+* standalone Composition Function testing and rendering without the need to access a Kubernetes cluster running Crossplane
+* visual dependency mapping across Compositions, Composite Resources and Managed Resources
+
+## Installing the CLI
+
+The Crossplane CLI is a single standalone binary with no external dependencies.
+
+{{<hint "note" >}}
+Install the Crossplane CLI on a user's computer. 
+
+The Crossplane CLI is independent of Kubernetes and doesn't require a Crossplane
+pod.
+{{< /hint >}} 
+
+To download the latest version for your CPU architecture with the Crossplane
+install script.
+
+```shell
+curl -sL "https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh" | sh
```

Comment:
Should we document the `XP_VERSION` and `XP_CHANNEL` variables that control which version of the CLI is downloaded and installed?